build.cmd

```diff
@@ -13,12 +13,17 @@ if not defined MSBUILDCUSTOMPATH (
     set MSBUILDCUSTOMPATH=MSBuild.exe
 )
 
+:: Check for a runtime host. If not defined, do not use a host
+if not defined RUNTIMEHOST (
+    set RUNTIMEHOST=""
+)
+
 echo ** MSBuild Path: %MSBUILDCUSTOMPATH%
 echo ** Building all sources
 
 :: Call MSBuild
-echo ** "%MSBUILDCUSTOMPATH%" "%~dp0build.proj" /verbosity:minimal /nodeReuse:false /fileloggerparameters:Verbosity=diag;LogFile="%~dp0msbuild.log" %*
-"%MSBUILDCUSTOMPATH%" "%~dp0build.proj" /verbosity:minimal /nodeReuse:false /fileloggerparameters:Verbosity=diag;LogFile="%~dp0msbuild.log" %*
+echo ** "%RUNTIMEHOST%" "%MSBUILDCUSTOMPATH%" "%~dp0build.proj" /verbosity:minimal /fileloggerparameters:Verbosity=diag;LogFile="%~dp0msbuild.log" %*
+"%RUNTIMEHOST%" "%MSBUILDCUSTOMPATH%" "%~dp0build.proj" /verbosity:minimal /fileloggerparameters:Verbosity=diag;LogFile="%~dp0msbuild.log" %*
```

Comment:
Does this work if you want to build with full-framework MSBuild and RUNTIMEHOST isn't defined (and then feeds in empty quotes)?